src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import java.util.Map;
+
+public interface Storage {
+    public void add(String fruit, int quantity);
```

Comment:
According to the encapsulation principle, interface methods are public by default, so there's no need to explicitly declare them as public. You can remove the 'public' modifier.